app/views/issues/index/_map.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @project and @project.module_enabled?(:gtt) %>
 
-<fieldset id="location" class="collapsible collapsed">
-<legend class="icon icon-collapsed"><%= l(:field_location) %></legend>
+<fieldset id="location" class="collapsible">
+  <legend class="icon icon-expended"><%= l(:field_location) %></legend>
```

Comment:
@dkastl (CC: @smellman, @iosefa)  
I modified the default icon to expended related with this.

About Firefox collapse issue, we need to merge `develop` branch's webpack improvement.